.changeset/seven-bikes-attend.md

```diff
@@ -0,0 +1,7 @@
+---
+"@fluidframework/test-driver-definitions": major
```

Comment:
Since this package is now deleted, you need to attach the changeset to the new package name.

```suggestion
"@fluid-internal/test-driver-definitions": major
```